services/IoT/applications/libraries/python.md

```diff
@@ -363,22 +359,22 @@ Applications can use the ``getOrganizationDetails()`` method to retrieve the det
         print("ERROR [" + e.httpcode + "] " + e.message)
 ```
 
-For information about the request & response model and the HTTP status code, refer to the Organization Configuration section of the [{{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html).
+For information about the request & response model and the HTTP status code, see the Organization Configuration section of the [{{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html).
 
 
 ## Bulk device operations
 {: #bulk_device_ops}
 
 Applications can use bulk operations to get, add, or remove multiple devices simultaneously.
 
-For information about the list of query parameters, the request & response model and http status code, see the Bulk Operations section of the [{{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html#!/Bulk_Operations/).
+For information about the list of query parameters, the request & response model, and http status code, see the Bulk Operations section of the [{{site.data.keyword.iot_short_notm}} API](https://docs.internetofthings.ibmcloud.com/swagger/v0002.html#!/Bulk_Operations/).
 
 
 ### Retrieving device information
 
-Bulk device information can be retrieved using the ``getAllDevices()`` method. This method retrieves information on all registered devices in the organization. Each request can contain a maximum of 512KB.
+Bulk device information can be retrieved by using the ``getAllDevices()`` method. This method retrieves information on all registered devices in the organization. Each request can contain a maximum of 512KB.
 
-The response contains parameters required by the application to retrieve the dictionary *results* from the response to get the array of devices returned. Other parameters in the response are required to make further calls, for example, the ``_bookmark`` element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark returned in the response and provide it on the request for the next page. Repeat until the end of the result set indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, otherwise the results are undefined.
+The response contains parameters that are required by the application to retrieve the dictionary *results* from the response to get the array of devices returned. Other parameters in the response are required to make further calls, for example, the ``_bookmark`` element can be used to page through results. Issue the first request without specifying a bookmark, then take the bookmark that is returned in the response and provide it on the request for the next page. Repeat until the end of the result set is indicated by the absence of a bookmark. Each request must use exactly the same values for the other parameters, otherwise the results are undefined.
```

Comment:
Fixed.

Clarified with dev. This means to get the array of devices that is returned. Changed here and elsewhere in this topic.